megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -311,6 +313,11 @@ public MovementDisplay(final ClientGUI clientgui) {
         buttons = new HashMap<>((int) (MoveCommand.values().length * 1.25 + 0.5));
         for (MoveCommand cmd : MoveCommand.values()) {
             String title = Messages.getString("MovementDisplay." + cmd.getCmd());
+            if ((clientgui != null) &&
+                    (title.equals(Messages.getString("MovementDisplay." + MoveCommand.MOVE_UNJAM.getCmd()))) &&
```

Comment:
This string comparison looks like it could be replaced by 
cmd == MoveCommand.MOVE_UNJAM